utils/src/main/java/com/cloud/utils/PropertiesUtil.java

```diff
@@ -34,6 +34,10 @@
 public class PropertiesUtil {
     private static final Logger s_logger = Logger.getLogger(PropertiesUtil.class);
 
+    public static String getDefaultApiCommandsFileName() {
+        return "commands.properties";
+    }
```

Comment:
Why is this method not defined in a common class/interface of the authenticator mechanism?